src/lib.rs

```diff
@@ -168,6 +171,167 @@ pub fn build_from_docker(
     Ok((file_output, measurements))
 }
 
+/// Extract signature section from EIF and parse the signing certificate
+fn get_certificate_info(eif_path: String) -> NitroCliResult<SignCertificateInfo> {
```

Comment:
Same let's group them under EifReader.

Reply:
I will refactor around this new structure.